apps/etterlatte-trygdetid/src/main/kotlin/trygdetid/regler/BeregnTrygdetid.kt

```diff
@@ -175,11 +176,19 @@ val opptjeningsDatoer: Regel<TrygdetidGrunnlagMedAvdoedGrunnlag, Pair<LocalDate,
         finnFelt = { Pair(it.foedselsDato, it.doedsDato) },
     )
 
+val norskPoengaar: Regel<TrygdetidGrunnlagMedAvdoedGrunnlag, Int?> =
+    finnFaktumIGrunnlag(
+        gjelderFra = TRYGDETID_DATO,
+        beskrivelse = "Hent overstyrt norsk poengaar",
```

Comment:
Kanskje noe slik som: _Finner antall poengår i Norge for overstyring av norske TT perioder_ er mer beskrivende?